raster/src/test/scala/geotrellis/raster/io/geotiff/OverviewStrategySpec.scala

```diff
@@ -0,0 +1,59 @@
+package geotrellis.raster.io.geotiff
```

Comment:
could you generate a header for tests? via `test:headerCreate`

Reply:
It is broken now, created an issue https://github.com/locationtech/geotrellis/issues/3201